cmd/fleetctl/api.go

```diff
@@ -205,3 +206,101 @@ func clientConfigFromCLI(c *cli.Context) (Context, error) {
 	}
 	return cc, nil
 }
+
+// apiCommand fleetctl api [options] uri
+// -F, --field <key=value>
+// Add a typed parameter in key=value format
+// -H, --header <key:value>
+// Add a HTTP request header in key:value format
+// -X, --method <string> (default "GET") 
+// The HTTP method for the request
+func apiCommand() *cli.Command {
+	var (
+		flField string
+		flHeader string
+		flMethod string
+	)
+	return &cli.Command{
+		Name:      "api",
+		Usage:     "Run an api command by uri",
+		UsageText: `fleetctl api [options] [url]`,
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:        "F",
+				EnvVars:     []string{"FIELD"},
```

Comment:
Should we have env vars equivalent for those? UX-wise, it seems unlikely that you want to store those (F, H, X) in an env var for multiple calls?